src/main/scala/org/broadinstitute/dsde/workbench/sam/dataAccess/PostgresAccessPolicyDAO.scala

```diff
@@ -1353,7 +1357,7 @@ class PostgresAccessPolicyDAO(protected val writeDbRef: DbReference, protected v
       userId: WorkbenchUserId,
       samRequestContext: SamRequestContext
   ): IO[Iterable[ResourceIdWithRolesAndActions]] =
-    readOnlyTransaction("listUserResourcesWithRolesAndActions", samRequestContext) { implicit session =>
+    readOnlyTransactionReadReplica("listUserResourcesWithRolesAndActions", samRequestContext) { implicit session =>
```

Comment:
do you want to do this for all usages of UserResourcesQuery?

Reply:
Not sure. The query that's been blowing up the DB seems to be the one with `as inherited` in it. Being as surgical as possible seems like the right move here.

Reply:
I vote that given where we are with this, we isolate this one major offender first.  80/20 rule.  We might be able to make things a lot better at some point, but for now we just need to treat the most severe problem.